json-rpc/src/methods.rs

```diff
@@ -103,10 +103,30 @@ async fn get_transactions(
 
     let mut result = vec![];
 
+    let all_events = if include_events {
+        txs.events
+            .ok_or_else(|| format_err!("Storage layer didn't return events when requested!"))?
```

Comment:
nit: I think this should be `unwrap_or_else`, since `events` is an `Option<..>`

Reply:
no,  unwrap_or_else() will make it return an  Vec<ContractEvents> or an Err() , which we can't use  "?" to proporgate error upward anymore.